ocpmodels/models/base.py

```diff
@@ -18,6 +27,49 @@ def __init__(self, num_atoms=None, bond_feat_dim=None, num_targets=None):
     def forward(self, data):
         raise NotImplementedError
 
+    def get_graph_properties(self, data):
+        if self.use_pbc:
+            if self.otf_graph:
+                edge_index, cell_offsets, neighbors = radius_graph_pbc(
+                    data, self.cutoff, self.max_neighbors
+                )
+            else:
+                edge_index = data.edge_index
+                cell_offsets = data.cell_offsets
+                neighbors = data.neighbors
+
+            out = get_pbc_distances(
+                data.pos,
+                edge_index,
+                data.cell,
+                cell_offsets,
+                neighbors,
+                return_offsets=True,
+                return_distance_vec=True,
+            )
+
+            edge_index = out["edge_index"]
+            edge_dist = out["distances"]
+            distance_vec = out["distance_vec"]
+        else:
+            self.otf_graph = True
```

Comment:
Might be good to support a case where both `otf_graph` and `use_pbc` are `False` and we should read the graph from the `data` object. Would enable debugging models with manually constructed graphs.

Reply:
That makes sense, I can  also check if the `data` object has `edge_index` when `otf_graph=False` and if not log a warning but still let the run continue with `otf_graph=True`.

Reply:
Yep, that sounds good to me.